easybuild/easyconfigs/p/Python/Python-2.7.14-GCCcore-7.2.0-bare.eb.bak_20180327135007

```diff
@@ -0,0 +1,36 @@
+name = 'Python'
```

Comment:
@rubendibattista please remove this backup easyconfig

Reply:
Ops. Did not see it! Sry!

Reply:
Same here, still needs to be removed?